packages/docs/versioned_docs/version-2.0.0-alpha-3/protocol/06-trees.md

```diff
@@ -4,29 +4,31 @@ description: The tree structures that are used in UniRep protocol.
 
 # Trees
 
-Each attester has a separate version of these trees each epoch.
+Each attester has an updated version of each tree for each epoch.
```

Comment:
Is this rewording accurate?

Reply:
Sorta/no. We start with an empty tree each epoch